telegram/replykeyboardmarkup.py

```diff
@@ -48,12 +48,18 @@ class ReplyKeyboardMarkup(ReplyMarkup):
         selective (:obj:`bool`, optional): Use this parameter if you want to show the keyboard to
             specific users only. Targets:
 
-            1) Users that are @mentioned in the text of the Message object.
-            2) If the bot's message is a reply (has ``reply_to_message_id``), sender of the
-               original message.
+            1) Users that are @mentioned in the :attr:`~telegram.Message.text` of the
+               :class:`telegram.Message` object.
+            2) If the bot's message is a reply (has reply_to_message_id), sender of the original
```

Comment:
did you remove the monospace-formatting on purpose?

Reply:
nope, wanted to link it but then realized that we can't link arguments of methods. adding it back...